README.md

```diff
@@ -40,6 +40,11 @@ resource "aws_cloudfront_distribution" "site" {
 `project` - Name of the project that this site is for (default: `Unknown`)
 `environment` - Name of the environment this site is targeting (default: `Unknown`)
 `region`  - Name of the region where buckets should be created (default: `us-east-1`)
+`cors_allowed_headers` - Allowed CORS headers (default: `[]`)
```

Comment:
I believe that the default CORS configuration for S3 buckets is defined by:

```xml
<CORSConfiguration>
    <CORSRule>
        <AllowedOrigin>*</AllowedOrigin>
        <AllowedMethod>GET</AllowedMethod>
        <MaxAgeSeconds>3000</MaxAgeSeconds>
        <AllowedHeader>Authorization</AllowedHeader>
    </CORSRule>
</CORSConfiguration>
```

I think that we should have the module defaults align with those.

I also think that we should keep `project` and `environment` at the bottom of this list in the `README`.